search-service/src/test/resources/ngsild/expectations/beehive_with_multi_instance_incoming_temporal_values.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+  "id" : "urn:ngsi-ld:BeeHive:TESTC",
+  "type" : "BeeHive",
+  "incoming" : [ {
```

Comment:
I'm a bit worried about this representation because it severely breaks the existing one.

I was more thinking of something like (just a sample, I'm not saying it has to be exactly like that):

```
"incoming": [
    ["urn:ngsi-ld:Dataset:01234", 550.0,"2020-03-25T08:29:17.965206Z"], ["urn:ngsi-ld:Dataset:01234", 650.0,"2020-03-25T08:33:17.965206Z"]
]
```

(All the possible use-cases have to be considered, of course)

Reply:
You mean we add the datasetId to the values elements ?

Reply:
I think we should keep the other properties.
As a result it could be something like that:
```
"incoming" : [
other_instance,
{
    "type" : "Property",
    "observedBy" : {
      "type" : "Relationship",
      "object" : "urn:ngsi-ld:Sensor:IncomingSensor"
    },
    "datasetId" : "urn:ngsi-ld:Dataset:45678",
    "values" : [ ["urn:ngsi-ld:Dataset:45678", 487.0, "2020-03-25T08:29:17.965206Z"], ["urn:ngsi-ld:Dataset:45678", 698.0,"2020-03-25T08:33:17.965206Z"] ],
    "observedAt" : "2020-01-24T13:01:22.066Z"
  },
other_instance
] 
```

Reply:
Cf my comments in the specification document.

And so the temporal evolution of the `incoming` property should be represented like this (also cf 4.5.9 - Simplified temporal representation of an Entity):

```
  "incoming" : [ {
    "type" : "Property",
    "datasetId" : "urn:ngsi-ld:Dataset:01234",
    "values" : [ [550.0,"2020-03-25T08:29:17.965206Z"], [650.0,"2020-03-25T08:33:17.965206Z"] ]
  }, {
    "type" : "Property",
    "datasetId" : "urn:ngsi-ld:Dataset:45678",
    "values" : [ [487.0,"2020-03-25T08:29:17.965206Z"], [698.0,"2020-03-25T08:33:17.965206Z"] ]
  } ],
```